azerothcore/azerothcore-wotlk
# BREAKING CHANGE(Core/Config): Individual XP rate per bg

## Changes Proposed:
-  One rate per BG. This would be useful if we would like to encourage people to do certain BG.

## Tests Performed:

- Tested in-game.

## How to Test the Changes:


1. Add the new configs to worldserver.conf
2. Set GiveXPForKills = 1 
3. Put some random number to Rate.XP.BattlegroundKill (or leave it as it is) and the other options.
4. See that your exp should be variable depending on the BG.

## Known Issues and TODO List:


- [ ] Better description, really open to improvements there :P
